bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/ssl/UniFiTrustManagerProvider.java

```diff
@@ -25,6 +27,7 @@
  * @author Matthew Bowman - Initial contribution
  */
 // @Component // [wip] mgb: disabled due to issues with service order loading
```

Comment:
@mgbowman Do you think we can enable this component again?

Reply:
@mgbowman Did you see my comment?

Reply:
This required some functionality from 2.5.0-M1 and has been re-enabled in my 2.5.0-SNAPSHOT build. It will make it into master once my latest changes get merged.

Reply:
@mgbowman Could you please reference your PR for those latest changes here?

Reply:
@kaikreuzer I haven't opened a PR yet as I had to pause development for holidays (read: family problems)

As a reference, here's the commit on my dev branch where I re-enabled this

https://github.com/mgbowman/openhab2-addons/commit/e932d59afafd5ea34a10437220342ae86a8b502b#diff-bc220d094ef0c603e97e0dda828cc406L72-R74

Reply:
@mgbowman Thanks for the link. What is the best way to move forward? Shall I remove my changes in the UniFi binding and we will wait for your contribution?

Reply:
@mgbowman would you have time to respond to @cweitkamp question:

> What is the best way to move forward? Shall I remove my changes in the UniFi binding and we will wait for your contribution?

Reply:
@cweitkamp can you revert this file from the PR so we can merge the rest. Thanks!

Reply:
@martinvw Sure. Done.